verticaltabs.jsm

```diff
@@ -109,6 +109,7 @@ VerticalTabs.prototype = {
     }.bind(this);
 
     this.window.VerticalTabs = this;
+    this.removeTab = this.window.gBrowser.removeTab.bind(this.window.gBrowser);
```

Comment:
I think we should move the `.bind` call down to line 440-ish, where we call this.remove…